gpt4all-bindings/python/gpt4all/pyllmodel.py

```diff
@@ -121,7 +121,7 @@ class LLModel:
     model_name: str
         Model name
     """
-
+    buffer = bytearray()
```

Comment:
`buffer` should probably be an instance attribute, not a class attribute (in `__init__()`).

If there are multiple `LLModel` instances at the same time, they would share this `buffer` which is not what you want.